contracts/MarketMaker.sol

```diff
@@ -32,7 +37,7 @@ contract MarketMaker is Ownable, IERC1155TokenReceiver {
     Stages public stage;
     enum Stages {
```

Comment:
A nitpick, and maybe a little outside the scope here, but since you're touching the `Stages` enum anyway, I'll add this to the mix:

I think enum type names (or rather type names in general) are typically expressed as singular (as opposed to plural) nouns. How this would look: `Stage stage` refers to a thing called `stage` which *is* a `Stage`.